content/en/blog/_posts/2023-11-1-Gateway-API-Future.md

```diff
@@ -0,0 +1,257 @@
+
+Notably, the `request` timeout encompasses the `backendRequest` timeout. Hence, the value of `backendRequest` should never exceed the value of the `request` timeout.
+
+The ability to configure these timeouts adds a new layer of reliability to your Kubernetes services.
+Whether it's ensuring client requests are processed within a specified timeframe or managing backend service communications, Gateway API's Timeouts offer the control and predictability you need.
+
+To get started, you can define timeouts in your HTTPRoute Rules using the Timeouts field, specifying their type as Duration.
+A zero-valued timeout (`0s`) disables the timeout, while a valid non-zero-valued timeout should be at least 1ms.
+
+Here's an example of setting request and backendRequest timeouts in an HTTPRoute:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: timeout-example
+spec:
+  parentRefs:
+  - name: example-gateway
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /timeout
+    timeouts:
+      request: 10s
+      backendRequest: 2s
+    backendRefs:
+    - name: timeout-svc
+      port: 8080
+```
+
+In this example, a `request` timeout of 10 seconds is defined, ensuring that client requests are processed within that timeframe.
+Additionally, a 2-second `backendRequest` timeout is set for individual requests from the Gateway to a backend service called timeout-svc.
+
+These new HTTPRoute Timeouts provide Kubernetes users with more control and flexibility in managing network communications, helping ensure a smoother and more predictable experience for both clients and backends. For additional details and examples, refer to the official timeouts API documentation.
+
+## Gateway Infrastructure Labels
+
+While Gateway API providers a common API for different implementations, each implementation will have different resources created under-the-hood to apply users intent.
+This could be configuring cloud load balancers, creating in cluster Pods and Services, or more.
+
+While the API has always provided an extension point -- `parametersRef` in `GatewayClass` -- to customize implementation specific things, there was no common core way to express common infrastructure customizations.
+
+Gateway API v1.0 paves the way for this with a new `infrastructure` field on the `Gateway` object, allowing customization of the underlying infrastructure.
+For now, this starts small with two critical fields: labels and annotations.
+When these are set, any generated infrastructure will have the provided labels and annotations set on them.
+
+For example, I may want to group all my resources for one application together:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: hello-world
+spec:
+  infrastructure:
+    labels:
+      app.kubernetes.io/name: hello-world
+```
+
+In the future, we are looking into more common infrastructure configurations, such as resource sizing.
```

Comment:
Comment applies to all sections, but would be helpful to have something like "for more information, refer to the docs for this feature" with a link to relevant documentation or GEP.